src/psij/executors/lsf.py

```diff
@@ -10,6 +10,7 @@
 from psij.executors.batch.batch_scheduler_executor import (
     BatchSchedulerExecutor,
     BatchSchedulerExecutorConfig,
+    check_status_exit_code,
```

Comment:
Would save you some imports if it was a method inside BatchSchedulerExecutor. It's only used in its subclasses after all.

Reply:
Yeah that's fine, on the other hand I think it's fine the way it is too since it doesn't need a reference to the class. I don't really want to rebase/force-push just for that.